src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -2576,12 +2576,20 @@ public override BoundNode VisitSizeOfOperator(BoundSizeOfOperator node)
         public override BoundNode VisitStackAllocArrayCreation(BoundStackAllocArrayCreation node)
         {
             VisitRvalue(node.Count);
+
+            if (node.InitializerOpt != null && !node.InitializerOpt.Initializers.IsDefault)
+            {
+                foreach (var element in node.InitializerOpt.Initializers)
```

Comment:
Please add braces on foreach to match rest of the file.

Reply:
this file is not consistent re braces.

Reply:
Nit: please add braces on foreach

---
In reply to: [166783444](https://github.com/dotnet/roslyn/pull/24249#discussion_r166783444) [](ancestors = 166783444)